spec/lib/submission_builder/state_return_spec.rb

```diff
@@ -131,4 +127,23 @@
       end
     end
   end
+
+  states_requiring_1099rs = StateFile::StateInformationService.active_state_codes.excluding(["nj", "ny"]) # why is nj excluded?
```

Comment:
Since you are basing this off of the 1099G tests, NJ was excluded there because NJ does not tax unemployment payments and therefore does not need the 1099G builder. It's fine to leave them excluded here for now, but I expect for 1099Rs they will want to be included. We can ask them to make that update if they wish. (but should probably delete that comment)

Reply:
@mmazanec22 @jachan @aloverso just flagging for ya'll that we're not testing 1099R inclusion in the XML document for NJ & the specs will need to be tweaked once you include them in your own xmls (same story for 1099G as well)